packages/create-effect-app/src/Cli.ts

```
@@ -249,7 +249,7 @@ function createTemplate(config: TemplateConfig) {
     // Handle user preferences for Nix flakes
     if (!config.projectType.withNixFlake) {
       yield* Effect.forEach(
-        [".envrc", "flake.lock", "flake.nix"],
+        [".envrc", "flake.nix"],
```

This effect was failing because `flake.lock` doesn't exist, so we're not going past this code in this case. I feel like this is not the best fix; we should try to avoid this kind of situation in the future.

I guess we could pass `{ force: true }` to `fs.remove` to ignore the exception if the path does not exist. Or handle it at the effect level and use something like `Effect.ignore/ignoreLogged`. Not sure if that's the best solution though, open to any better idea ?

---

@tatchi - thank you! I think either solution is fine, but I feel like I would probably use `Effect.ignore` for this case.

Either way, this PR looks good for this particular issue. If you want to add `Effect.ignore`s in the relevant places as part of this PR let me know. Otherwise also let me know if you're happy with this as is and I'll get this merged 👍

---

Should we basically add `Effect.ignore` to every place where we use `fs.ignore` ?

---

I think we can start with just adding it to places where we attempt to remove files.